### DISCUSSION TITLE
Login problem when jumping from PC to PC, location and VPN

### DISCUSSION THREAD

**Discussion by filipdot** [nextcloud/all-in-one#3234] [Questions] [answered]
Hi, i'm have some problem with login. It's hard to use NC because of that. If you can give any hint what i could do wrong i'll be grateful.

I am log in.
Connect to VPN or move to another Pc in same network or move to other location - cannot log in using user name or email. Sometime clearing cookies help but not always. Sometime incognito in browser help but not always. Sometimes going to TALK help before timeout.
Sometimes i can log in but i see timeout message like this (in Polish):
![obraz](https://github.com/nextcloud/all-in-one/assets/85545543/64ed54bf-295e-4233-8bbf-99514a25cf7c)
It say: "There were a problem to read website, reconnecting in 2 seconds." and "You are not log in" -> WHAT? I just did it.
What i check?
Password - it's good, i have 2 entry in my password manager 1 using username, 1 using email with same password.
Different PC - i log in at home on PC, in office on Laptop, in others location on Laptop. Problem is every place.
Update NC - i have problem from 26, now on 27.0.2.
Restart docker containers.

### Steps to reproduce - I'm not sure it can be easy reproduce.
1. Log in
2. Change your PC/LAPTOP public IP (VPN, mobile access point, go home/work)
3. Try log in and fail to do so.

### Expected behavior 
I log in and mind my stuff.
### Actual behavior 
Not log in.

### Host OS 
Ubuntu 22.04 LTS on Proxmox VE 7.15
Using NGinx Proxy Manager for proxy.

#### Nextcloud AIO version 
Nextcloud AIO v7.0.0

#### Current channel 
You are running the [latest] channel.
#### Other valuable info 
Log's when we talk about login info.

![obraz](https://github.com/nextcloud/all-in-one/assets/85545543/e4c811ec-3fc9-4d4b-bed8-961ea187c738)

**Comment by szaimen** [comment]
Hm... Does it work if you disable the suspicious_login app?

**Comment by filipdot** [comment]
Turned off, testing for few days. I'll post if it helps.

**Reply by filipdot** [reply]
testing and still happend few times

![20230907_100928](https://github.com/nextcloud/all-in-one/assets/85545543/f339aada-8924-474e-b61f-6ee3c86bbdc1)


after cleaning cookies i can login after 2-3 try

**Reply by szaimen** [reply, marked as answer]
Unfortunately I cannot reproduce your issue so it is most likely caused by your network and VPN.

**Reply by filipdot** [reply]
how it efect only nextcloud?

After error's i try to log in using user name and it's take me back to login screen with my user but with no message. Login not fail but not sucessfull? Same if i try using email. And after few try i'm block for to many try.
If i clear cookies it's fail on 1 try, but have sucess in second one.
Maybe it's mishandling of cookies by nextcloud?

**Reply by szaimen** [reply]
Can you follow https://github.com/nextcloud/all-in-one/discussions/2475?

**Reply by filipdot** [reply]
i try but still i get errors and it's something with token. After clearing cockie i can log in.

[core] Info: Tried to log in but could not verify token
GET /ocs/v2.php/apps/user_status/api/v1/statuses/Filip
from 172.23.0.1 at 2024-01-10T08:01:33+00:00

{"reqId":"aIA2IgxOqTYywfSsy2GO","level":1,"time":"2024-01-10T08:01:33+00:00","remoteAddr":"172.23.0.1","user":"--","app":"core","method":"GET","url":"/ocs/v2.php/apps/user_status/api/v1/statuses/Filip","message":"Tried to log in but could not verify token","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:121.0) Gecko/20100101 Firefox/121.0","version":"27.1.5.1","data":{"app":"core","user":"Filip"},"id":"659e4f063f325"}
![obraz](https://github.com/nextcloud/all-in-one/assets/85545543/f991f486-46fe-4b07-a340-e0f777e7eb14)

when i try to log out i can't do it i get
"Access denied"
"CSFR check failed"
I didn't find any issues on that matter.
![obraz](https://github.com/nextcloud/all-in-one/assets/85545543/9acaa65b-9ca4-479c-9e66-26b541f86f41)

I was on my laptop yesterday, and open up today without changing network or anything. It's on the same place/eth/wifi etc.

**Comment by szaimen** [comment]
Moving to discussions while waiting

**Comment by Phonemyat242** [comment]
https://github.com/nextcloud/all-in-one/discussions/3234#discussioncomment-6810850
